hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/om/helpers/OmKeyInfo.java

```diff
@@ -488,6 +501,8 @@ public Builder addMetadata(String key, String value) {
 
     public Builder addAllMetadata(Map<String, String> newMetadata) {
       metadata.putAll(newMetadata);
+
+      setHSync(metadata.containsKey(OzoneConsts.HSYNC_CLIENT_ID));
```

Comment:
@smengcl this is what I was referring to.

Reply:
Got it. Is this a placeholder for now?

The goal is to prevent other clients from opening the file while the first client hasn't closed the hsync'ed file yet correct?

Reply:
Actually we want to enable this behavior such that any client can forcefully close open files (HBase expects this)